service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/ControlledAiNotebookHandlerTest.java

```diff
@@ -73,6 +72,6 @@ public void generateInstanceId_userIdTooLong_trim() {
     int maxNameLength = MAX_INSTANCE_NAME_LENGTH;
 
     assertEquals(maxNameLength, instanceId.length());
-    assertTrue(instanceId.equals(instanceId.substring(0, maxNameLength)));
+    assertEquals(instanceId, instanceId.substring(0, maxNameLength));
```

Comment:
nit: I think the args are backwards here

Reply:
Done